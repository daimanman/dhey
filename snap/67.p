{"C":1,"Duration":201461840,"EndTime":1537176597986885200,"Err":"","FileId":"3","InputFileName":"file","Method":"POST","N":1,"P":null,"Payload":"","QH":{},"Remark":"压力测试","ReqNums":1,"StartTime":1537176597785423400,"Status":1,"TaskId":67,"Type":"FILE","Url":"http://192.168.1.192:9091/api/v1/oss","Z":""}