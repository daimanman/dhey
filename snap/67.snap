{"AvgTotal":0.2004638,"Fastest":0.2004638,"Slowest":0.2004638,"Average":0.2004638,"Rps":4.9884318265941285,"AvgConn":0.0009956,"AvgDNS":0,"AvgReq":0.0010047,"AvgRes":0.0019889,"AvgDelay":0.1964746,"ConnMax":0,"ConnMin":0,"DnsMax":0,"DnsMin":0,"ReqMin":0.0010047,"ReqMax":0.0010047,"ResMin":0.0010047,"ResMax":0.0019889,"DelayMax":0.1964746,"DelayMin":0.1964746,"Lats":[0.2004638],"ConnLats":[0],"DnsLats":[0],"ReqLats":[0.0010047],"ResLats":[0.0019889],"DelayLats":[0.1964746],"Total":200463800,"ErrorDist":{},"StatusCodeDist":{"200":1},"SizeTotal":0,"SizeReq":0,"NumRes":1,"LatencyDistribution":[{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0},{"Percentage":0,"Latency":0}],"Histogram":[{"Mark":0.2004638,"Count":1,"Frequentcy":1},{"Mark":0.2004638,"Count":0,"Frequentcy":0},{"Mark":0.2004638,"Count":0,"Frequentcy":0},{"Mark":0.2004638,"Count":0,"Frequentcy":0},{"Mark":0.2004638,"Count":0,"Frequentcy":0},{"Mark":0.2004638,"Count":0,"Frequentcy":0},{"Mark":0.2004638,"Count":0,"Frequentcy":0},{"Mark":0.2004638,"Count":0,"Frequentcy":0},{"Mark":0.2004638,"Count":0,"Frequentcy":0},{"Mark":0.2004638,"Count":0,"Frequentcy":0},{"Mark":0.2004638,"Count":0,"Frequentcy":0}],"TaskId":67,"Duration":0.2004638}